src/logger/loggerApi.c

```diff
@@ -1352,6 +1347,133 @@ int api_setCanConfig(struct Serial *serial, const jsmntok_t *json)
     return API_SUCCESS;
 }
 
+static void _json_can_mapping(struct Serial *serial, CANMapping *mapping, int more)
```

Comment:
Yes, I think we can standardize on put. changing to json_put_can_mapping

we have some places where we use the term 'send' - those can be refactored later.